.github/workflows/main.yml

```diff
@@ -20,9 +20,8 @@ jobs:
         node-version: ${{ matrix.node-version }}
         cache: npm
     - run: |
-          wget -q https://packages.erlang-solutions.com/erlang/debian/pool/esl-erlang_21.3.8.17-1~ubuntu~bionic_amd64.deb
-          sudo apt install --allow-downgrades ./esl-erlang_21.3.8.17-1~ubuntu~bionic_amd64.deb || true
-          sudo apt --fix-broken install
+          wget -q https://packages.erlang-solutions.com/erlang/debian/pool/esl-erlang_25.2.3-1~ubuntu~jammy_amd64.deb
+          sudo apt install --allow-downgrades ./esl-erlang_25.2.3-1~ubuntu~jammy_amd64.deb
```

Comment:
OTP 24 is usually used around the environments, I checked with @hanssv and CLI CI and OTP 24 should sufficient